.eslintignore

```diff
@@ -0,0 +1,5 @@
+/node_modules
```

Comment:
This can be merged into `.eslintrc.js` via the `ignorePatterns`.

We're also already ignoring [`/node_modules` & `/vendor`](https://github.com/ackama/eslint-config-ackama/blob/main/index.js#L15-L25)